EventsExpress/ClientApp/src/components/inventory/InventoryList.js

```diff
@@ -94,69 +128,134 @@ class InventoryList extends Component {
         });
     }
 
+    onWillTake = inventar => {
+        const data = {
+            eventId: this.props.eventId,
+            userId: this.props.user.id,
+            inventoryId: inventar.id,
+            quantity: Number(inventar.willTake)
+        }
+
+        if (inventar.isWillTake) {
+            this.props.want_to_take(data);
+        } else {
+            this.props.edit_users_inventory(data);
+        }
+
+        this.setState({
+            disabledEdit: false
+        });
+    }
+
+    onWillNotTake = inventar => {
+        const data = {
+            eventId: this.props.eventId,
+            userId: this.props.user.id,
+            inventoryId: inventar.id
+        }
+
+        this.props.delete_users_inventory(data);
+    }
+
+    renderInventoryItem = item => {
+        const { user, usersInventories } = this.props;
+        let isMyEvent = this.props.event.owners.find(x => x.id === user.id) != undefined;
+        return (
+            <div className="row p-1 d-flex align-items-center" key={item.id}>
+                {item.isEdit && isMyEvent && 
+                    <OwnerEditItemForm
+                        onSubmit={this.onSubmit} 
+                        onCancel={this.onCancel}
+                        unitOfMeasuringState={this.props.unitOfMeasuringState}
+                        alreadyGet={usersInventories.data.reduce((acc, cur) => {
```

Comment:
This code is duplicated. Consider extracting it to a separate method or local function.